README.md

```diff
@@ -90,7 +89,18 @@ Or, with a poe task:
 
 It is recommended to run Kyma Companion with Poetry because it activates and uses its virtual environment if not activated yet.
 
-> [!NOTE] You cannot run it with Python directly.
+As an alternative, you can run the application with the following command directly via Python, using [`uvicorn`](https://www.uvicorn.org/) instead of `FastAPI`:
+
+   ```bash
+   python src/main.py
+   ```
+
+If you need auto-reloading, you can add the corresponding key word argument in the `src/main.py`:
+
+   ```python
+   if __name__ == "__main__":
+       uvicorn.run(app, host="0.0.0.0", port=8000, reload=True)
+   ```
```

Comment:
```suggestion
If you need auto-reloading, you need to pass the `--reload` argument:
`python src/main.py --reload`
For IDEs (Pycharm, VS Code), you need to pass this argument in the run/debug configuration.
```

Reply:
I tried the `python src/main.py` which is not working when you curl the applcaiont. 
Shouldn't you use some like this `uvicorn main:app --reload`?